packages/lesswrong/themes/createThemeDefaults.js

```diff
@@ -51,6 +51,14 @@ const createLWTheme = (theme) => {
         linkUnderlinePosition: "72%",
         ...body2FontSize
       },
+      // used by h3
+      // TODO: We could increase the numbers of the displayN up by one, but
+      // that sounds merge-conflict-y
+      display1a: {
```

Comment:
Hmm. This smells pretty weird to me. I think I prefer actually going up to Display4 as top-level title (which seems actually more like what we should have been using all along, setting that style to what is currently display3), and incrementing each h-level up one. Not confident though.

Would like a second opinion from @Discordius or @jimrandomh tho

Reply:
As an additional wrinkle, the next major version of Material UI removes/renames all of these.

Reply:
Yeah, I chatted briefly with Oli (and I think you) about that at the time I was first reviewing this, asking about when we were going to do that migration. (It may not have been clear at the time why I was asking that, apologies)

Do you think the fact that that migration is coming some day is a point towards or against this particular version? It feels a bit hacky to me, but not sure it's worth making JP change the PR if we're basically going to switch to a whole other system.

Reply:
The work of renaming is not that much, it will just explode the number of changes, introducing merge conflicts everywhere. Don't worry about the work on my end though. It's like 30 minutes.

Reply:
nod. ping me on slack when it's done and I'll try to review it ASAP so the conflicts don't multiply.

Reply:
SG (The "let me know when it's done" seems like there's afore-knowledge that it's been agreed upon that I should do it, whereas I really consider it a call with positives and negatives on LW's end, and was waiting for a yay-or-nay.) [Thanks for continuing to be on top of this though.]

Reply:
Oli and I chatted and it seemed fine to leave it mostly as-is, but change "display1a" to "display0" which is a bit more clear about what order they're supposed to come it.